tc_driver/dycore.jl

```diff
@@ -397,8 +399,7 @@ function compute_gm_tendencies!(
     aux_en_f = TC.face_aux_environment(state)
     aux_up = TC.center_aux_updrafts(state)
     aux_bulk = TC.center_aux_bulk(state)
-    ρ_f = aux_gm_f.ρ
```

Comment:
is it not used ?

Reply:
Just not inside the scope of this function, it seems